core/pkg/resourcehandler/k8sresources.go

```diff
@@ -147,6 +148,29 @@ func (k8sHandler *K8sResourceHandler) GetClusterAPIServerInfo() *version.Info {
 	return clusterAPIServerInfo
 }
 
+// set  namespaceToNumOfResources map in report
+func setMapNamespaceToNumOfResources(allResources map[string]workloadinterface.IMetadata, sessionObj *cautils.OPASessionObj) {
+	mapNamespaceToNumberOfResources := make(map[string]int)
+	for _, resource := range allResources {
+		if obj := workloadinterface.NewWorkloadObj(resource.GetObject()); obj != nil {
+			ownerReferences, err := obj.GetOwnerReferences()
+			if err == nil {
+				// if object is highest level and belong to namespace (except Job), add to map
+				if len(ownerReferences) == 0 {
+					if ns := resource.GetNamespace(); ns != "" {
+						if obj.GetKind() != "Job" {
+							mapNamespaceToNumberOfResources[ns]++
+						}
+					}
+				}
+			} else {
```

Comment:
Use warning or info not error. 
If this is an error we should return an error.
And explain that it will not be counted.